Tools/machines/perlmutter-nersc/install_gpu_dependencies.sh

```diff
@@ -130,9 +130,9 @@ python3 -m pip install --upgrade matplotlib
 python3 -m pip install --upgrade yt
 # install or update WarpX dependencies such as picmistandard
 python3 -m pip install --upgrade -r $HOME/src/warpx/requirements.txt
-python3 -m pip install cupy-cuda11x  # CUDA 11.7 compatible wheel
+python3 -m pip install cupy-cuda12x  # CUDA 12 compatible wheel
 # optional: for libEnsemble
 python3 -m pip install -r $HOME/src/warpx/Tools/LibEnsemble/requirements.txt
 # optional: for optimas (based on libEnsemble & ax->botorch->gpytorch->pytorch)
-python3 -m pip install --upgrade torch  # CUDA 11.7 compatible wheel
+python3 -m pip install --upgrade torch  # CUDA 12 compatible wheel
```

Comment:
This one I'm just assuming will be okay...

Reply:
yes, they updated their default: https://pytorch.org/get-started/locally/